File: src/test/samples/invalidSamples/invalidSample.kt
fun main() {
    // #region FirstRegion
    val x = 42
    //#endregion

// #endregion Invalid end boundary
// #region Invalid start boundary

    //#region Second Region  
    class MyClass {
        // #region   InnerRegion   
        fun myMethod() {}
        //  #endregion   ends InnerRegion 

        // #region
        fun myMethod2() {}
               //#endregion 
    }
    // #endregion
}

File: src/test/samples/invalidSamples/invalidSample.vb
#Region "FirstRegion"
Dim x As Integer = 42
#End Region

  #End Region "Invalid end boundary"
  #Region "Invalid start boundary"

#Region "Second Region"
Public Class MyClass
    #Region      "InnerRegion"  
    Public Sub MyMethod()
    End Sub
    #End Region     ends InnerRegion  

    #Region
    Public Sub MyMethod2()
    End Sub
    #End Region
End Class
#End Region
